src/js/utils/translation/en.ts

```diff
@@ -0,0 +1,60 @@
+import { TranslateDictionary } from '../translation-helper.js';
+
+export const En: TranslateDictionary = {
+  tooltips: {
+    enterAirplay: 'Start airplay',
```

Comment:
this is starting to look good. some thoughts that came to mind.

1. this grouping / categories (tooltips, nouns, ...) is not needed and even the indexing with special keys as well
  let's make this a simple dict like
```js
export const En = {
  'Start airplay': 'Start airplay',
  ...
```
 - this reduces byte size slightly because of gzip compression 
 - it's easier for translation because the literal english word is on the left side as key.